index.html

```diff
@@ -404,6 +404,178 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
+    Each IoT project is different, each customer uses a different framework.
+    
+    Ben: agree with all points of Sebastian, not all profiles should impose storage size and bandwidth limits.
+    
+    Cristiano: Disagree with limit storage and bandwidth requirements and Use finite (maximum) resources.
+    
+    Ege: agree on the requirement, we need to determine what we exactly mean with "complexity"
+    
+    Oracle: Complexity applies to all potential adopters, i.e. things, gateways, edge, cloud
+    -->
+    
+    <h3>No Ambiguities, select single choice</h3>
+    <!-- Supporters: Oracle, Fujitsu, Intel, Ben, Cristiano, Ege -->
+    <p>
+      Get rid of ambiguities, i.e. clarify specifications to define interpretation of a TD and behavior of the thing and
+      consumer.
+    </p>
+    <p>
+      Examples are the choice of properties vs. actions, use of PUT or POST for HTTP, observe protocols.
+    </p>
+    <!--
+    Notes: 
+    Intel: If something is really ambiguous, it needs to be updated in the TD, if there are multiple choices, 
+    Profile can narrow down to one choice. 
+    Ben: broadly support the requirement, however requiring that all profiles adopt the same ontology. 
+    Choice of actions vs. properties should be done in the TD.
+    This will probably create new requirements for the TD spec.
+    -->
+    
+    <h3>Developer guidance</h3>
+    <!-- Supporters: Fujitsu, Siemens, Intel, Oracle, Ben, Cristiano-->
+    <p>
+      A profile should help define what needs to be implemented. This requirement also includes behavioral goals and
+      recommendations about best practice for the implementation of Consumers and Things.
+    </p>
+    <h3>Multiple profiles (mechanism)</h3>
+    <!-- Supporters: Intel, Siemens, Ben, Cristiano, Hitachi -->
+    <p>
+      The mechanism used to indicate that a TD satisfies a profile should be general enough to indicate
+      the TD satisfies the requirements for multiple profiles.
+    </p>
+    <!--
+      Some people are concerned about fragmentation, if multiple profiles would be defined.
+      However this requirement is about the mechanism to identify the profile in use.
+  
+    Discussion: 
+    Does this mean a thing can support multiple profiles? 
+    TD already supports multiple profiles Profiling mechanism is described in the profile spec, 
+    may need to be polished
+    -->
+    
+    <h3>Composable profiles</h3>
+    <!-- Proposer: Intel, Hitachi, Siemens, Ben(*)-->
+    <p>
+      It should be possible to combine multiple profiles both for production and consumption:
+    </p>
+    <p>
+      It should be possible to indicate that a consumer can ingest TDs that satisfy one or more profiles, even if each TDs
+      individually only satisfies one profile. For example, a Smart Building may need to use both "home" devices and
+      "industrial" devices. A gateway consuming TDs should be able to ingest TDs designed for both the home and industrial
```

Comment:
"home" vs. "industrial" devices is a bad example, please remove or change (e.g. to constrained vs. unconstrained).